docs/root/intro/arch_overview/load_balancing.rst

```diff
@@ -88,6 +88,17 @@ panic threshold is 50%. This is :ref:`configurable <config_cluster_manager_clust
 runtime. The panic threshold is used to avoid a situation in which host failures cascade throughout
 the cluster as load increases.
 
+.. _arch_overview_load_balancing_priority_levels:
+
+Priority levels
+------------------
+
+During load balancing, Envoy will generally only consider hosts configured at the highest priority
+level.  For each EDS ``LocalityLbEndpoint``, an optional priority may also be specified.  Currently
+the failover mechanics for routing from one priority level to another are fairly simplistic: a given
+priority level will be used until it has zero healthy hosts, at which ponint it will hard fail to
```

Comment:
typo: "ponint" (I also see some "over enthusiastic" spaces ;).

Reply:
Oh carp, the whitespace checker doesn't run against MD files, does it?  Well at least I have something to do in meetings tomorrow...